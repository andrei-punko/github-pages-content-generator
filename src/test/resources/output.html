<one>
Page name
<two>
<h1>Page title</h1><hr/>
<p align="justify">
<b id="q1">Смирение</b>&nbsp;<a href="#q1">⬅</a><br/>
Бог гордым противится, а смиренным дает благодать<br/>
</p>
<h2>Молитва</h2><hr/>
<p align="justify">
Молитва - с вниманием, благоговением и покаянием<br/>
</p>
<p align="justify">
При выборе пути жизненного стоит использовать смирение<br/>
</p>
<p align="justify">
<b id="q2">Об осуждении</b>&nbsp;<a href="#q2">⬅</a><br/>
<a href="https://azbyka.ru/otechnik/Dorofej/dushepoleznye-pouchenija-i-poslanija/">https://azbyka.ru/otechnik/Dorofej/dushepoleznye-pouchenija-i-poslanija/</a><br/>
Помышляющий сердцем о множестве своих грехов никогда не захочет из соблазна другого сделать предмет для своего разговора. Судить человека соблазнившегося есть знак гордости, а гордым Бог противится. Напротив, кто каждый час готовится дать ответ о своих грехах, нескоро поднимет голову, чтобы рассматривать чужие ошибки<br/>
</p>
<p align="justify">
Самое первое, что надо делать в практической борьбе с грехом осуждения, — перестать рассказывать и слушать про чужие грехи. Это надо запретить себе делать категорически. И второе — не позволять даже мысленно осуждать другого человека, потому что все наши грехи начинаются в мыслях. Для этого лучше всего, едва приходит помысл осуждения, отсекать его молитвой Иисусовой, то есть, мысленно произнести: «Господи Иисусе Христе, Сыне Божий, помилуй мя, грешного».<br/>
</p>
<p align="justify">
<b id="q3">Ленивый Singleton</b>&nbsp;<a href="#q3">⬅</a><br/>
<pre>
public class Singleton {		// Synchronized Accessor
    private static Singleton instance;

    public static synchronized Singleton getInstance() {
        if (instance == null) {
            instance = new Singleton();
        }
        return instance;
    }
}
</pre>
</p>
<p align="justify">
<b id="q4">Xml example</b>&nbsp;<a href="#q4">⬅</a><br/>
<pre>
&lt;one&gt;
  &lt;two&gt;
  &lt;/two&gt;
&lt;/one&gt;
</pre>
</p>
<p align="justify">
<b id="q5">Bulleted list</b>&nbsp;<a href="#q5">⬅</a><br/>
= one<br/>
Bla-bla<br/>
</p>
<p align="justify">
= two<br/>
== sub two-1<br/>
One more line<br/>
== sub two-2<br/>
</p>
<p align="justify">
= three<br/>
== sub three-1<br/>
== sub three-2<br/>
</p>
<p align="justify">
= four<br/>
</p>

</two>
</one>
