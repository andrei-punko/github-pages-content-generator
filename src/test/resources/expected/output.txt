
<p align="justify">
<b>Смирение</b></br>
Бог гордым противится, а смиренным дает благодать</br>
молитва - с вниманием, благоговением и покаянием</br>
при выборе пути жизненного стоит использовать смирение</br>
</p>
<p align="justify">
<b>Об осуждении</b></br>
Помышляющий сердцем о множестве своих грехов никогда не захочет из соблазна другого сделать предмет для своего разговора. Судить человека соблазнившегося есть знак гордости, а гордым Бог противится. Напротив, кто каждый час готовится дать ответ о своих грехах, нескоро поднимет голову, чтобы рассматривать чужие ошибки</br>
</p>
<p align="justify">
Самое первое, что надо делать в практической борьбе с грехом осуждения, — перестать рассказывать и слушать про чужие грехи. Это надо запретить себе делать категорически. И второе — не позволять даже мысленно осуждать другого человека, потому что все наши грехи начинаются в мыслях. Для этого лучше всего, едва приходит помысл осуждения, отсекать его молитвой Иисусовой, то есть, мысленно произнести: «Господи Иисусе Христе, Сыне Божий, помилуй мя, грешного».</br>
</p>
